playwright/tests/organize/journeys/journey-instance/page.spec.ts

```diff
@@ -104,9 +104,7 @@ test.describe('Journey instance page', () => {
       );
 
       await Promise.all([
-        page.waitForResponse(
-          `**/orgs/${KPD.id}/journey_instances/${ClarasOnboarding.id}`
-        ),
+        page.waitForResponse((res) => res.request().method() == 'PATCH'),
```

Comment:
I don't think this is necessary, although I do prefer this because it's more clear that we're waiting for the `PATCH` and not a `GET` that may be initiated by react-query after a cache invalidation.